client/routes.jsx

```diff
@@ -51,6 +51,15 @@ const routes = (store) => {
       <Route path="/:username/sketches" component={IDEView} />
       <Route path="/:username/assets" component={IDEView} />
       <Route path="/:username/account" component={forceToHttps(AccountView)} />
+      <Route path="/assignment/:classroom_id/:assignment_id" component={IDEView} />
```

Comment:
The conventions I try to stick to when creating routes are from [Rails](http://guides.rubyonrails.org/routing.html). Therefore, follow those conventions, this route should be `/classrooms/:classroom_id/assignments/:assignment_id`